src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_COUNT = 5;
+    private static final String COLOR_DEFAULT = "WHITE";
+    private static final int RADIUS_DEFAULT = 10;
+    private Random random = new Random();
+    private String randomColor;
+    private Figure figure;
+
+    private Circle getRandomCircle() {
```

Comment:
private methods should go after public ones

Reply:
private meth put after public